chemicaltoolbox/chembl/chembl.py

```diff
@@ -0,0 +1,102 @@
+from chembl_webresource_client.new_client import new_client
+import argparse
+
+def open_file(filename):
+    with open(filename) as f:
+        return f.read().split('\n')[0]
+
+def get_smiles(res):
+    """
+    Get a list of SMILES from function results
+    """ 
+    smiles = []
```

Comment:
Should that be a set to exclude duplicates?

Reply:
changed